app/components/post/index.js

```diff
@@ -52,6 +56,15 @@ function makeMapStateToProps() {
         let isFirstReply = true;
         let isLastReply = true;
         let commentedOnPost = null;
+        const channel = getChannel(state, {id: post.channel_id}) || {};
+        const canPost = haveIChannelPermission(
+            state,
+            {
+                channel: post.channel_id,
+                team: channel.team_id,
+                permission: Permissions.CREATE_POST,
```

Comment:
what server version was this added? do we need to check for the server version before checking for the permission?

Reply:
This is not a new permission so I assumed we don't need to check. It was part of the original set of permissions released about 2 years ago, but was never enforced on the web and mobile clients.